apps/searchSummary/src/contexts/SearchContext.tsx

```diff
@@ -227,41 +228,44 @@ export const SearchContextProvider = ({ children }: Props) => {
       if (initialSearchResponse.response.length > 0) {
         const startTime = Date.now();
         try {
```

Comment:
I think we need to call `setSummarizationResponse("")` to clear the last summary, otherwise there's a moment where it's still visible even after the user has submitted a query.